bos-umbrel-install.sh

```diff
@@ -6,11 +6,25 @@
 
 set -eo pipefail
 
+# Verify the network type 
+# that the user has set.
+network="mainnet"
```

Comment:
I would use local variables. This avoids polluting the global name space and inadvertently setting variables that may have significance outside the function.

I'll refactor and then merge :)
```suggestion
local network="mainnet"
```